### PR TITLE
Add documentation for v5.0

### PR DESCRIPTION
**Pull request by DL6ER**
Documentation updates related to the v5.0 release.

### CODE REVIEW

File: docs/database/gravity/lists.md [hunk 1]
```diff
@@ -14,16 +15,22 @@ Label | Type | Uniqueness enforced | Content
 `date_modified` | integer | No | Timestamp when domain was last modified, automatically updated when a record is changed
 `comment` | text | No | Optional field for arbitrary user comments
 
-## Blacklist Table (`blacklist`)
-The `blacklist` table contains all blacklisted domains. Just like the `whitelist` table, it has a few extra fields to store data related to a given domain such as the `enabled` state, the dates when the domain were added and when it has last been modified, and an optional comment.
+### RegEx Whitelisting Table (`regex_whitelist`)
```
Comment (Mcat12) [inline docs/database/gravity/lists.md]: For consistency, do we want to use `Regex` or `RegEx`? Up to now it seemed that `Regex` is the most common form.

File: docs/ftldns/regex/overview.md [hunk 2]
```diff
@@ -1,10 +1,17 @@
-A regular expression, or RegEx for short, is a pattern that can be used for building arbitrarily complex blocking rules in *FTL*DNS.
+A regular expression, or RegEx for short, is a pattern that can be used for building arbitrarily complex filter rules in *FTL*DNS.
 We implement the POSIX Extended Regular Expressions similar to the one used by the UNIX `egrep` (or `grep -E`) command.
 
 Our implementation is light and fast as each domain is only checked once for a match (if you query `google.com`, it will be checked against your RegEx. Any subsequent query to the same domain will not be checked again until you restart `pihole-FTL`).
 
-## How to use regular expressions for blocking
-*FTL*DNS reads in regular expression filters from the [`regex` database table](../../database/gravity/lists.md#regex-table-regex).
+## Hierarchy of regex filters in *FTL*DNS
+*FTL*DNS uses a specific hierarchy to ensure regex filters work as you expect them to. Whitelisting always  has higher propriety than blacklisting.
```
Comment (Mcat12) [inline docs/ftldns/regex/overview.md]:
```suggestion
*FTL*DNS uses a specific hierarchy to ensure regex filters work as you expect them to. Whitelisting always has priority over blacklisting.
```

File: docs/index.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ description: Brief overview of Pi-hole
 last_updated: Thur May 02 01:00:00 2019 UTC
 ---
 <p align="center">
-<a href="https://pi-hole.net"><img src="https://pi-hole.github.io/graphics/Vortex/Vortex_with_text.png" width="150" height="255" alt="Pi-hole"></a><br/>
+<a href="https://pi-hole.net"><img src="images/Vortex_with_text.png" width="150" height="255" alt="Pi-hole"></a><br/>
```
Comment (Mcat12) [inline docs/index.md]: @dschaper is this ok with you? I remember there was some initiative before to centralize the images used on various readmes and documentation sites.

Comment (dschaper) [reply docs/index.md]: If you want to include the graphics repo as a submodule to this repo and then pull directly, that would work.

**Pull request merged**